### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by JeroenVdb** [joeferner/node-http-mitm-proxy#263]
[Recently](https://github.com/joeferner/node-http-mitm-proxy/commit/b901de9d274d115fad4d880e55df2e36aa20a047) node-forge dependency was updated which fixes a high vulnerability. Could this update be release to npm in the near future? Of is there another recommended path to take to fix this vulnerability in upstream libraries?

**Comment by Apollon77** [comment]
Yes @joeferner please do so! ALso because if the "2year -> 1year" certificate length change! Requred urgently!

**Comment by Apollon77** [comment]
@joeferner Any chance? ;-)

**Comment by joeferner** [comment]
done
